### DISCUSSION TITLE
How to check rook-ceph block storage utilization at pod level?

### DISCUSSION THREAD

**Discussion by chintha-v** [rook/rook#9963] [Q&A]
Hi All,
We are using rook-ceph block storage as default storage class. Multiple pods are consuming PVCs from it. Using "ceph osd df", we are able to see OSD level storage consumption. But when I consolidated "df -kh" value of all pods which are using this storage class, it is not matching and more less compared to "ceph osd df" output. We are not able to trace which one is consuming that extra space. 
Ceph commands (ceph -s, ceph osd df, ceph df) are giving stats at OSD level but not giving who is consuming those OSDs and how much.
How to get information at pod level or at some resource level which one is consuming and how much space from rook-ceph-block storage class?
Please help.
Thank you so much in advance!

**Comment by travisn** [comment]
@chintha-v Try [enableRBDStats: true](https://rook.github.io/docs/rook/v1.8/ceph-pool-crd.html#pool-settings) on the pools where you want to gather the metrics.
